sources/packages/api/src-sql/StudentScholasticStandings/Revert-scholastic-standing-approval-fields.sql

```diff
@@ -0,0 +1,24 @@
+-- Revert scholastic standing approval fields.
+-- Create approved_data.
+ALTER TABLE
+  sims.student_scholastic_standings
+ADD
+  COLUMN IF NOT EXISTS approved_data JSONB;
+
+COMMENT ON COLUMN sims.student_scholastic_standings.approved_data IS 'Dynamic form data that represents the final data revised by the Ministry.';
+
+-- Create assessed_by.
+ALTER TABLE
+  sims.student_scholastic_standings
+ADD
+  COLUMN IF NOT EXISTS assessed_by INT REFERENCES sims.users(id) ON DELETE CASCADE;
```

Comment:
`DELETE ` is there for the initial create migration

Reply:
It shouldn't be a concern on a rollback script IMO.

Reply:
I was mentioning about ON DELETE CASCADE which could be tricky sometimes. But not in this case.

Reply:
The rollback need put back in place the behavior that was before, if the CASCADE was there then it must be present on the rollback script to put the DB back to its previous state.
That said, we should use `ON DELETE SET NULL` to avoid the unintentional removal of absolutely everything that is related to the user like every single record that the user is present as some sort of audit user.
That said, I asked @abschwenker to check with DBAs if we can get rid of all CASCADING options. It is still pending an answer.